packages/core/crates/core-crypto/src/routing.rs

```diff
@@ -18,7 +18,7 @@ pub const fn header_length<S: SphinxSuite>(
     additional_data_relayer_len: usize,
     additional_data_last_hop_len: usize,
 ) -> usize {
-    let per_hop = <S::P as Keypair>::Public::SIZE + SimpleMac::SIZE + additional_data_relayer_len;
+    let per_hop = <S::P as Keypair>::Public::SIZE + 1 + SimpleMac::SIZE + additional_data_relayer_len;
```

Comment:
I'd name this constant `1`, because upon reading it I have no idea what it is or why it is there.